source/Random.cpp

```diff
@@ -34,6 +36,11 @@ namespace {
 #endif
 }
 
+static bool normalBMCached = false;
+static double cachedBMNormal = std::numeric_limits<double>::infinity();
+static const double epsilon = std::numeric_limits<double>::epsilon();
+static const double two_pi = 2.0 * PI;
```

Comment:
`two_pi` seems like a better fit for the `pi.h` file.

Reply:
This comment seems slightly stale.  I switched over to using pi.h yesterday and didn't bother to declare two_pi anymore, I just used PI in sigma.

Reply:
No, disregard that.  I did use `PI` from `pi.h`, but I do declare `two_pi` in Random.  Seems like a reasonable suggestion.

Reply:
Fixed in cbdcdc0401e048de789d2cfe174d26ce2ed0c40e.